extensions/aws/tests/CMakeLists.txt

```diff
@@ -35,5 +35,6 @@ FOREACH(testfile ${AWS_INTEGRATION_TESTS})
     target_link_libraries(${testfilename} minifi-expression-language-extensions)
     MATH(EXPR AWS_TEST_COUNT "${AWS_TEST_COUNT}+1")
     add_test(NAME "${testfilename}" COMMAND "${testfilename}" WORKING_DIRECTORY ${TEST_DIR})
+    set_tests_properties("${testfilename}" PROPERTIES LABELS "aws")
```

Comment:
and here

Reply:
Same here, there was a strange leak directly from aws sdk, we should investigate further later